### ISSUE TITLE
Bonus

### ISSUE THREAD

**Issue by josephgupkim** [lltheis/codecademy-the.scoop#4]
Doesn't seem like there was an attempt at the bonus. As it wasn't mandatory it's no big deal but if I'm mistaken, please do let me know.

**Comment by lltheis** [comment]
Nope, unfortunately no attempt here right now. I'm barely staying afloat
with this class and two jobs unfortunately. Hoping to come back to it.
Laura
